modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -68,13 +67,32 @@ def allow_processing?
     end
 
     def va_gov_errored_claims
-      va_gov = ClaimsApi::AutoEstablishedClaim.select(:id, :transaction_id)
-                                              .where(created_at: @search_from..@search_to,
-                                                     status: 'errored', cid: '0oagdm49ygCSJTp8X297')
-                                              .group(
-                                                :id, :transaction_id
-                                              )
-      va_gov.group_by(&:transaction_id)
+      errored_claims = get_unique_errors
+      errored_claims.map { |record| record[0] } # rubocop:disable Rails/Pluck
+    end
+
+    def get_unique_errors
+      begin
+        last_day = ClaimsApi::AutoEstablishedClaim
+                   .where(created_at: 24.hours.ago..1.hour.ago,
+                          status: 'errored', cid: '0oagdm49ygCSJTp8X297')
+
+        last_hour = ClaimsApi::AutoEstablishedClaim
+                    .where(created_at: 1.hour.ago..Time.zone.now,
+                           status: 'errored', cid: '0oagdm49ygCSJTp8X297')
+
+        day_trans_ids = last_day.pluck(:transaction_id)
+
+        errored_claims = last_hour.find_all do |claim|
+          day_trans_ids.exclude?(claim[:transaction_id])
+        end
```

Comment:
👍

also unlikely, but if there are no errors in `last_day`, we will hit an error on line 84.